sfge/src/main/java/com/salesforce/graph/symbols/apex/ApexValue.java

```diff
@@ -220,16 +220,22 @@ public boolean isNotNull() {
      * null or it is a an uninitialized variable which will be initialized to null by the system.
      */
     public boolean isNull() {
-        if (ValueStatus.UNINITIALIZED.equals(status)) {
-            // Uninitialized values are initialized to null by the compiler
-            return true;
-        } else if (ValueStatus.INDETERMINANT.equals(status)) {
-            return false;
+        boolean returnValue =
```

Comment:
This was the bug that didn't count null constraints in indeterminant strings. Fixed the order of checks.